### PR TITLE
Restructure API calls

### PR DESCRIPTION
**Pull request by brionmario**
## Purpose

- Move all API calls to a `api` folder.
- Separate out the API calls in the `kfone-websites` `api/index` to different contexts for better readability.

## Goals

- Fixes https://github.com/asgardeo-samples/asgardeo-enterprise-ciam-sample/issues/4
- Fixes https://github.com/asgardeo-samples/asgardeo-enterprise-ciam-sample/issues/2

### CODE REVIEW

File: src/apps/kfone-customer-support/.env.example [hunk 1]
```diff
@@ -7,4 +7,8 @@ REACT_APP_ASGARDEO_CALLBACK_URL=<CALLBACK URL AFTER A SUCCESSFUL AUTHENTICATION>
 REACT_APP_CHOREO_CLIENT_ID=<CONSUMER ID TAKEN FROM ASGARDEO CONSOLE>
 REACT_APP_BASE_API_ENDPOINT=<BASE URL OF CHOREO HOSTED API>
 REACT_APP_CHOREO_ORGANIZATION=<ORGANIZATION OF CHOREO HOSTED API>
-REACT_APP_CHOREO_TOKEN_ENDPOINT=<CHOREO TOKEN ENDPOINT>
+
+# STS Token Endpoint is the endpoint to obtain security tokens from a security token service.
+# You can obtain this value from your identity provider.
+# Ex: For Choreo, https://sts.choreo.dev/oauth2/token
+REACT_APP_STS_TOKEN_ENDPOINT='<your_sts_token_endpoint_here>'
```
Comment (ashendes) [inline src/apps/kfone-customer-support/.env.example:14]: Are quotes needed here? And better to have the description similar to the other configs.

File: src/apps/kfone-customer-support/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (ashendes) [inline src/apps/kfone-customer-support/.eslintrc.js:2]: Is this the correct license header?

File: src/apps/kfone-customer-support/.env.example (hunk 1 above)
Comment (brionmario) [reply src/apps/kfone-customer-support/.env.example:14]: Planning to change the description of the other configs as well.
Quotes are not really needed but seem to give better highlighting.
Will try to remove it.

**Review by ashendes — APPROVED** [review]

File: src/apps/kfone-customer-support/.env.example (hunk 1 above)
Comment (brionmario) [reply src/apps/kfone-customer-support/.env.example]: Fixed with https://github.com/asgardeo-samples/asgardeo-enterprise-ciam-sample/pull/13/commits/452eaec7b871daf10e5910933d58e0bb77fa297c

File: src/apps/kfone-customer-support/.eslintrc.js (hunk 2 above)
Comment (brionmario) [reply src/apps/kfone-customer-support/.eslintrc.js]: This is the one we are using in `WSO2` front end repos.

**Review by ashendes — APPROVED** [review]

**Pull request merged**